records_mover/records/sources/base.py

```diff
@@ -117,6 +118,16 @@ def move_to_records_directory(self,
         directory while converting their format."""
         pass
 
+    @abstractmethod
+    def temporary_unloadable_directory_scheme(self) -> str:
+        """Which URL scheme will be used to create the temporary location to fill in."""
+        pass
+
+    @contextmanager
+    def temporary_unloadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
+        # TODO: write some docs
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker